src/opera/pge/rtc_s1/rtc_s1_pge.py

```diff
@@ -285,6 +339,34 @@ def _rtc_geotiff_filename(self, inter_filename):
 
         return f"{rtc_filename}_{polarization}.tif"
 
+    def _browse_filename(self, inter_filename):
+        """
+        Returns the file name to use for the PNG browse image product produced
+        by this PGE.
+
+        The filename for RTC metadata products consists of:
+
+            <RTC filename>_BROWSE.png
+
+        Where <RTC filename> is returned by RtcS1PostProcessorMixin._rtc_filename().
+
+        Parameters
+        ----------
+        inter_filename : str
+            The intermediate filename of the output product to generate
+            a filename for. This parameter may be used to inspect the file
+            in order to derive any necessary components of the returned filename.
+
+        Returns
+        -------
+        browse_filename : str
```

Comment:
I don't know what the convention is where an anonymous variable is invovled, maybe just put the type?